buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+import java.util.Properties
+
+repositories {
+    google()
+    maven(url = "https://plugins.gradle.org/m2/")
+    mavenCentral()
+    jcenter()
+}
+
+plugins {
+    java
+    `kotlin-dsl`
+}
+
+val properties = Properties().apply {
+    load(rootProject.file("../versions.properties").inputStream())
+}
+
+dependencies {
+    implementation(gradleApi())
+    implementation("com.android.tools.build:gradle:${properties["gradlePlugin"] as String}")
+    implementation("org.jacoco:org.jacoco.core:0.8.5")
+    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.71")
```

Comment:
use kotlin version here too?

Reply:
The same problem as the AGP with access scope is not been shared with any easy. The Kotlin version that Gradle used for the `.gradle.kts` also trails by quite a bit, so, not always having this version in sync should be fine.